src/tests/access/test_accesscontrol.cairo

```diff
@@ -447,31 +464,44 @@ fn test_default_admin_role_is_its_own_admin() {
 // Helpers
 //
 
-fn assert_event_role_revoked(role: felt252, account: ContractAddress, sender: ContractAddress) {
-    let event = utils::pop_log::<AccessControlComponent::Event>(ZERO()).unwrap();
-    let expected = AccessControlComponent::Event::RoleRevoked(
-        RoleRevoked { role, account, sender }
-    );
-    assert!(event == expected);
-    utils::assert_no_events_left(ZERO());
-}
-
-fn assert_event_role_granted(role: felt252, account: ContractAddress, sender: ContractAddress) {
-    let event = utils::pop_log::<AccessControlComponent::Event>(ZERO()).unwrap();
-    let expected = AccessControlComponent::Event::RoleGranted(
-        RoleGranted { role, account, sender }
-    );
-    assert!(event == expected);
-    utils::assert_no_events_left(ZERO());
-}
-
-fn assert_event_role_admin_changed(
-    role: felt252, previous_admin_role: felt252, new_admin_role: felt252
-) {
-    let event = utils::pop_log::<AccessControlComponent::Event>(ZERO()).unwrap();
-    let expected = AccessControlComponent::Event::RoleAdminChanged(
-        RoleAdminChanged { role, previous_admin_role, new_admin_role }
-    );
-    assert!(event == expected);
-    utils::assert_no_events_left(ZERO());
+#[generate_trait]
+impl AccessControlSpyHelpersImpl of AccessControlSpyHelpers {
+    fn assert_event_role_revoked(
+        ref self: EventSpy,
+        from_address: ContractAddress,
```

Comment:
Why `from_address` instead of `contract` like in the other tests? Do you think `from_address` is more clear?

Reply:
Changed to `contract` for consistency